packages/matchbox/src/components/Tooltip/tests/Tooltip.test.js

```diff
@@ -81,6 +60,7 @@ describe('Tooltip', () => {
 
   it('should pass through system props', () => {
     const wrapper = subject({ bg: 'blue', fontSize: '400', pr: '500' });
+    wrapper.find('button').simulate('mouseOver');
```

Comment:
Should we test for focus here, too, or just in the cypress tests?

Reply:
I don't think theres a way to test focus in jest, I think the cypress test is enough though